<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookAddDialog</class>
 <widget class="QWidget" name="BookAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,0,20">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">image: url(:/resources/images/MartinIdenCover.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="titleLineEdit">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="categoriesLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Montserrat</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Category(ies):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="categoriesComboBox">
           <property name="font">
            <font>
             <family>Montserrat</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <item>
            <property name="text">
             <string>Action</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Flash</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Something</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Anything</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>For</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Comedy</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="categoriesListWidget">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="authorsTitle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Montserrat</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Autrhor(s):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="authorsComboBox">
           <property name="font">
            <font>
             <family>Montserrat</family>
             <pointsize>9</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <item>
            <property name="text">
             <string>Action</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Flash</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Something</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Anything</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>For</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Comedy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jack London</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Family Guy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alexander Pushkin</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QListWidget" name="authorsListWidget">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="publicationDateLabel">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Publication Date:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleLineEdit</tabstop>
  <tabstop>categoriesComboBox</tabstop>
  <tabstop>authorsComboBox</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>categoriesListWidget</tabstop>
  <tabstop>authorsListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
